src/cli/generate.rs

```diff
@@ -81,5 +82,13 @@ mod tests {
             .args(&["g", "zomes/bubblechat", "rust"])
             .assert()
             .success();
+
+         Command::main_binary()
+            .unwrap()
+            .current_dir(&tmp.path())
+            .args(&["g", "zomes/zubblebat", "asm"])
```

Comment:
I'm not sure about the `asm`. I know this is AssemblyScript but `asm` could also mean `asmjs`